src/app/modal-classroom/modal-classroom-dialog.component.html

```diff
@@ -1,9 +1,9 @@
 <h1 mat-dialog-title >Chai Spies</h1>
 <div mat-dialog-content>
```

Comment:
Moving forward, we need to figure out how to display the day at a glance, I'm guessing this will be included in the next sprint?